doc/api/async_hooks.md

```diff
@@ -221,12 +221,13 @@ The following is a simple demonstration of `triggerId`:
 
 ```js
 const async_hooks = require('async_hooks');
```

Comment:
Well, they need not, but many readers will test the snippets, so I prefer to be consistent towards completeness, not implying (RE: empathy :)